packages/fxa-settings/package.json

```diff
@@ -16,16 +16,42 @@
     "stop": "pm2 stop pm2.config.js",
     "delete": "pm2 delete pm2.config.js",
     "storybook": "npm run build-postcss && start-storybook -p 6008 -s .storybook/design-guide --no-version-updates",
-    "test": "SKIP_PREFLIGHT_CHECK=true rescripts test"
+    "test": "SKIP_PREFLIGHT_CHECK=true rescripts test",
+    "test:coverage": "yarn test --coverage --watchAll=false"
   },
   "jest": {
     "resetMocks": false,
     "collectCoverageFrom": [
       "src/**/*",
       "!src/**/*.stories.*",
       "!src/**/mocks.*",
-      "!src/**/*.svg"
-    ]
+      "!src/**/*.svg",
+
+      "!src/index.*",
+      "!src/components/*(ScrollToTop|UnitRowRecoveryKey|PageAvatar|App)/*",
```

Comment:
Are we ignoring these because we need to increase coverage on them to reach the threshold set, and the idea is to file a follow up to increase coverage and remove this line later?

Same questions for `models/`, I think we would want coverage on at least the exported class `Account` right?

Reply:
Yeah, I was thinking these would need to improved one by one, because they had very low test coverage, and this would have really lowered the overall thresholds.

In my opinion, the models should not have logic in them, so there wouldn't be anything to test. Maybe that's a different maintenance item though.

Reply:
The models here perform operations such as making calls to the [auth client](https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/models/Account.ts#L540:L540) and storing/retrieving data from [apollo](https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/models/hooks.ts#L37), so I don't think it would hurt to have coverage count against them.

Reply:
Can we pull those ops into a different space so that our models are pure?